lib/logcat.js

```diff
@@ -1,118 +1,80 @@
-"use strict";
+import { SubProcess } from 'teen_process';
+import log from './logger';
```

Comment:
right, I'm saying I think we could just have two prefixes, one for adb and one for logcat. from the user's perspective it doesn't matter that logcat is part of the adb package